library/wizard/src/modules/Confirm.rb

```diff
@@ -55,7 +54,7 @@ def main
 
     # Confirm hardware detection (only in manual installation)
     # @param [String] class hardware class (network cards)
-    # @param [String] icon_name name of the icon. If nil, generic hardware icon will be used
+    # @param [String] obsolete - just for compatibility
     # @return true on continue
     def Detection(class_, icon_name)
```

Comment:
you use it somewhere or rubocop should complain, that unused parameter should start with _